test/support_bundle/sb_interface.py

```diff
@@ -0,0 +1,138 @@
+import os
+import sys
+import argparse
+import subprocess
+import shlex
+import tarfile
+import time
+
+from sb_config import PV_CLAIM_LIST, SB_FILE_PATH, SB_TAG
+
+
+class SSPLBundleError(Exception):
```

Comment:
Remove SSPL. There is nothing called SSPL in CORTX for LC.

Reply:
Ack.